integration-test/testcases/compression.basic.spec.ts

```diff
@@ -0,0 +1,118 @@
+import { GenericContainer, StartedTestContainer, Wait } from 'testcontainers';
+import { ExasolDriver, websocketFactory } from '../../src/lib/sql-client';
+import { RandomUuid } from 'testcontainers/dist/uuid';
+import { DOCKER_CONTAINER_VERSION } from '../runner.config';
+import { ExasolPool, LogLevel, Logger } from '../../src';
+
+export const basicCompressionTests = (name: string, factory: websocketFactory) =>
+  describe(name, () => {
+    const randomId = new RandomUuid();
+    let container: StartedTestContainer;
+    jest.setTimeout(7000000);
+    let schemaName = '';
+
+    beforeAll(async () => {
+      container = await new GenericContainer(DOCKER_CONTAINER_VERSION)
+        .withExposedPorts(8563, 2580)
+        .withPrivilegedMode()
+        .withDefaultLogDriver()
+        .withReuse()
+        .withWaitStrategy(Wait.forLogMessage('All stages finished'))
+        .start();
+    });
+
+    beforeEach(() => {
+      schemaName = 'TEST_SCHEMA' + randomId.nextUuid();
+    });
+
+    it('Exec and fetch', async () => {
+
+      const setupClient = createClient(factory,container,false,LogLevel.Off)
```

Comment:
Would it be possible to use only a single client with compression enabled? That would automatically verify that `execute()` also works with compression.

Reply:
I'm probably not getting what you're saying here but isn't that exactly what's going on? `setupClient `has compression set to false. whereas `clientWithCompression `has compression enabled

Reply:
Ah, I see what you mean. Well I can just enable compression for the setup client.